.ci/scripts/install-tools.bat

```diff
@@ -1,15 +1,25 @@
 set GOPATH=%WORKSPACE%
 set MAGEFILE_CACHE=%WORKSPACE%\.magefile
-set PATH=%WORKSPACE%\bin;C:\ProgramData\chocolatey\bin;%PATH%
+set PATH=%WORKSPACE%\bin;C:\ProgramData\chocolatey\bin;C:\tools\mingw64\bin;%PATH%
```

Comment:
Where the installation of the gcc is done.

NOTE: this could be removed as soon as the packer images have been deployed with support to gcc.